shared/src/commonMain/kotlin/org/hyperskill/app/sentry/domain/interactor/SentryInteractor.kt

```diff
@@ -56,4 +58,14 @@ class SentryInteractor(
             sentryManager.finishTransaction(transaction, throwable)
         }
     }
+
+    // Conforming to AnalyticEventMonitor
+
+    override fun analyticDidReportEvent(event: AnalyticEvent) {
+        println("SentryInteractor: analyticDidReportEvent: $event")
```

Comment:
This approach will not work for Android. We need to use special Logger as planned previously.

Reply:
> This approach will not work for Android.

Why this will not work? It's already worked as expected, you can see Breadcrumbs in the Sentry console. For example this "[Event](https://sentry.int.stepik.org/organizations/sentry/discover/hyperskill-android:1ee0378a16e146b49a287cad4a018330/?environment=production-debug&field=id&field=user.display&field=transaction.duration&field=trace&field=timestamp&name=step-feature-screen-remote-data-loading&project=12&query=transaction.duration%3A%3C15m+event.type%3Atransaction+transaction%3Astep-feature-screen-remote-data-loading&sort=-transaction.duration&statsPeriod=1h&yAxis=count%28%29)".

Reply:
I meen`println("log message")` will not print anything into console on Android

Reply:
OK, you wrote a comment for a commit https://github.com/hyperskill/mobile-app/pull/331/commits/ad3a32183cec4f4f8d0b8d9d4be80e4707177a7f, not for a [final PR changes](https://github.com/hyperskill/mobile-app/pull/331/files).